test/features/dataset_schema.feature

```diff
@@ -0,0 +1,43 @@
+@schema_metadata
+Feature: Dataset Schema
+
+    Scenario Outline: Add new dataset metadata fields for default data schema validation
+        Given "<User>" as the persona
+        When I log in
+        And I visit "/dataset/new"
+
+        And I should see an element with xpath "//label[text()="Default data schema"]"
```

Comment:
"Then" would make more sense than "And" since it's an assertion.

Reply:
+

Reply:
Will be fixed and pushed by Alex shortly.